nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -310,8 +315,180 @@ def __init__(self, seeds, radius=None, mask_img=None, allow_overlap=False,
         self.memory = memory
         self.memory_level = memory_level
 
+        # Parameters for reporting
+        self.reports = reports
+        self.report_id = -1
+        self._report_content = dict()
+        self._report_content['description'] = (
+            'This reports shows the regions '
+            'defined by the spheres of the masker.')
+        self._report_content['warning_message'] = None
+
         self.verbose = verbose
 
+    def generate_report(self, displayed_spheres=10):
```

Comment:
By contrast, I find that setting displayed _spheres to 10 by default is really arbitrary and surprising. I would default to 'all'